docker-images/worker/Dockerfile

```diff
@@ -47,7 +51,6 @@ RUN pip install click \
 
 RUN pip install git+https://github.com/dask/dask@0a7fe8de2eef643f797afe5fda4c1b0bda8ad03d\
                 git+https://github.com/dask/distributed@7909f27d591612537bb23974d089869edb11d097\
-                msgpack \
```

Comment:
Yes, the same error arises if I leave this in. My thinking here was to copy the already-working notebook image.